stat/distmv/normal.go

```diff
@@ -277,6 +277,38 @@ func (n *Normal) Rand(x []float64) []float64 {
 	return x
 }
 
+// ScoreInput returns the score function with respect to the input of the
```

Comment:
Because the score function https://en.wikipedia.org/wiki/Score_(statistics) is the derivative with respect to the parameters, and this is with respect to the inputs.